src/Experimental/isomorphism.jl

```diff
@@ -120,12 +130,20 @@ false
 ### See also
 [`has_induced_subgraphisomorph`](@ref), [`has_isomorph`](@ref), [`count_subgraphisomorph`](@ref), [`all_subgraphisomorph`](@ref)
 """
-function has_subgraphisomorph(g1::AbstractGraph, g2::AbstractGraph, alg::IsomorphismAlgorithm=VF2();
-                                 vertex_relation::Union{Nothing, Function}=nothing,
-                                 edge_relation::Union{Nothing, Function}=nothing)::Bool
-    has_subgraphisomorph(g1, g2, alg;
-        vertex_relation=vertex_relation,
-        edge_relation=edge_relation)
+function has_subgraphisomorph(
+    g1::AbstractGraph,
+    g2::AbstractGraph,
+    alg::IsomorphismAlgorithm = VF2();
+    vertex_relation::Union{Nothing,Function} = nothing,
+    edge_relation::Union{Nothing,Function} = nothing,
+)::Bool
+    has_subgraphisomorph(
+        g1,
+        g2,
+        alg;
+        vertex_relation = vertex_relation,
+        edge_relation = edge_relation,
+    )
```

Comment:
I don't understand the purpose of this function at all. Why do we even need it?

Reply:
Well this function test for the existence of a subgraph isomorphism. I think it is valuable to have something like that. I am actually not sure how it is currently implement, but in a perfect case, it would return a boolean withouth temporarily allocation any mapping.

Reply:
My point is that it's simply another method that doesn't have defaults for the kwargs. It's the same as the one above it.